src/apollo.js

```diff
@@ -0,0 +1,22 @@
+import fetch from "cross-fetch"
+import { ApolloClient, InMemoryCache, HttpLink } from "@apollo/client"
+
+const GITHUB_TOKEN_READ_ONLY =
+  process.env.CONTEXT === "production"
+    ? process.env.GITHUB_TOKEN_READ_ONLY_PROD
+    : process.env.CONTEXT === "deploy-preview"
+    ? process.env.GITHUB_TOKEN_READ_ONLY_STAGING
+    : process.env.GITHUB_TOKEN_READ_ONLY_DEV
+
+const client = new ApolloClient({
+  link: new HttpLink({
+    uri: `https://api.github.com/graphql`,
+    fetch,
+    headers: {
+      Authorization: `Bearer ${GITHUB_TOKEN_READ_ONLY}`,
```

Comment:
I believe we need to update this to a `GATSBY_...` ENV variable so it's accessible from the fronted. See my original PR here: https://github.com/ethereum/ethereum-org-website/pull/2296/files#diff-9bd41b9894ddb2787f97e601c440c1cf7b1018f8a52498fbec796bdc422efcf4